pom.xml

```diff
@@ -656,43 +669,6 @@
     </pluginManagement>
   </build>
   <profiles>
-    <profile>
-      <id>java9+</id>
-      <activation>
-        <jdk>[9,)</jdk>
-      </activation>
-      <build>
-        <pluginManagement>
-          <plugins>
-            <plugin>
-              <groupId>org.apache.maven.plugins</groupId>
-              <artifactId>maven-compiler-plugin</artifactId>
-              <executions>
-                <execution>
-                  <id>jdk9</id>
-                  <goals>
-                    <goal>compile</goal>
-                  </goals>
-                  <configuration>
-                    <release>9</release>
-                    <compileSourceRoots>
-                      <compileSourceRoot>${project.basedir}/src/main/java9</compileSourceRoot>
-                    </compileSourceRoots>
-                    <outputDirectory>${project.build.outputDirectory}/META-INF/versions/9</outputDirectory>
-                    <compilerArgs>
-                      <arg>--patch-module</arg>
-                      <arg>org.assertj.core=${project.build.outputDirectory}</arg>
-                      <arg>--module-version</arg>
-                      <arg>${project.version}</arg>
-                    </compilerArgs>
```

Comment:
Don't we need these arguments in the new configuration?

Reply:
No. 
It looks like `maven-compiler-plugin` add it.

When I inspect by `mvn compile -X` I have:

#### old pom

```
jdk 8

[DEBUG] -d .../assertj-core/target/classes 
-classpath ...
-sourcepath .../assertj-core/src/main/java:.../assertj-core/target/generated-sources/annotations: 
-s .../assertj-core/target/generated-sources/annotations 
-g -nowarn 
-target 1.8 
-source 1.8 
-encoding UTF-8

jdk 9

[DEBUG] -d .../assertj-core/target/classes/META-INF/versions/9 
-classpath .../assertj-core/target/classes: 
--module-path ...
-sourcepath .../assertj-core/src/main/java9:.../assertj-core/target/generated-sources/annotations: 
-s .../assertj-core/target/generated-sources/annotations 
-g -nowarn 
--release 9 
-encoding UTF-8 
--patch-module org.assertj.core=.../assertj-core/target/classes 
--module-version 3.19.1-SNAPSHOT 
--module-version 3.19.1-SNAPSHOT
```

#### new pom

```
jdk 8

[DEBUG] -d .../assertj-core/target/classes 
-classpath ...
-sourcepath .../assertj-core/src/main/java:.../assertj-core/target/generated-sources/annotations: 
-s .../assertj-core/target/generated-sources/annotations 
-g -nowarn 
--release 8 
-encoding UTF-8

jdk 9

[DEBUG] -d .../assertj-core/target/classes/META-INF/versions/9 
-classpath .../assertj-core/target/classes: 
--module-path ...
-sourcepath .../assertj-core/src/main/java9:.../assertj-core/target/generated-sources/annotations: 
-s .../assertj-core/target/generated-sources/annotations 
-g -nowarn 
--release 9 
-encoding UTF-8 
--patch-module org.assertj.core=.../assertj-core/target/classes 
--module-version 3.19.1-SNAPSHOT

```